### DISCUSSION TITLE
Can RSPAMD replace Scrollout?

### DISCUSSION THREAD

**Discussion by verus001** [rspamd/rspamd#3632] [General]
Hi,
Our e-mail server is an exchange server and we have an antispam gateway in front of that (Scrollout).
Can RSPAMD replace the scrollout gateway entirely so that we only have RSPAMD with the exchange server?
That is not so clear for the moment.
Thanks. Ph.

**Comment by vstakhov** [comment]
As far as I see, Scrollout is just a email gate with full MTA and everything hidden under the hood (presumably Postfix). Rspamd can apparently work in conjunction with a forwarding MTA, probably @andryyy has something similar in the Mailcow project.

**Reply by verus001** [reply]
If RSPAMD could configure their product with the same possibilities (combination to use directly with an Exchange server and everything under the hood), it would gain a lot of users for this product I guess.

**Reply by andryyy** [reply]
You can use it with Postfix or any other sane MTA. I cannot think of an Exchange setup that does not have any MTA for filtering/rewriting in front of it. :)
